data/tweaks/mtu-probing/metadata.yml

```diff
@@ -0,0 +1,3 @@
+description: "Enable MTU probing (fixes some applications like Ubisoft Connect)"
+type: toggle
+stability: 4
```

Comment:
maybe in the config format, each level should be represented as a string (ex. stable, devel), which gets parsed?